CONTRIBUTORS.md

```diff
@@ -8,6 +8,8 @@
  - Jon Maier
  - Gnu-Bricoleur
  - Brian Dunlay
+ - KyleMoran138
```

Comment:
You used generate_contributors.py to generate this? Otherwise, your addition will be overwritten the next time that script is run.

Reply:
It was generated when I ran `test.sh`. It isn't the exact output because it added my username twice to the file.

Reply:
If so, edit `generate_contributors.py` to exclude one of the usernames, whichever you prefer to omit =).